crates/wit/src/commands/publish.rs

```diff
@@ -55,31 +48,14 @@ impl PublishCommand {
         };
         let file_cache = FileCache::new(cache_dir(self.common.cache_dir)?).await?;
 
-        let url = find_url(
-            self.registry.as_deref(),
-            &config.registries,
-            warg_config.home_url.as_deref(),
-        )?;
-
-        let signing_key = if let Ok(key) = std::env::var("WIT_PUBLISH_KEY") {
-            Some(PrivateKey::decode(key).context(
-                "failed to parse signing key from `WIT_PUBLISH_KEY` environment variable",
-            )?)
-        } else {
-            None
-        };
-
         publish_wit_package(
             PublishOptions {
                 config: &config,
                 config_path: &config_path,
-                warg_config: &warg_config,
                 pkg_config,
                 cache: file_cache,
-                url,
-                signing_key,
+                url: self.registry.as_ref(),
```

Comment:
Nit but should we rename `url` field to `registry`, since it is [not a URL](https://github.com/bytecodealliance/wasm-pkg-tools/blob/main/crates/wasm-pkg-common/src/registry.rs#L11)